File: php/problem_9.php
<?php
function special_pythagorean_triplet($input)
{
  foreach (range(1, $input) as $a) {
    foreach (range(1, ($input - $a)) as $b) {
      $c = $input - $a - $b;
      if ($a ** 2 + $b ** 2 === $c ** 2) {
        return $a * $b * $c;
      }
    }
  }
}

$answer = 31875000;
echo special_pythagorean_triplet(1000) === $answer ? "true" : "false";

File: python/problem_9.py
special_pythagorean_triplet = [a*b*(1000-a-b) for a in range(1, 1000)
                               for b in range(1, 1000-a) if a**2+b**2 == (1000-a-b)**2][0]
answer = 31875000
print(True if special_pythagorean_triplet == answer else False)
